test/YesSql.Tests/NamingPolicyTests.cs

```diff
@@ -0,0 +1,89 @@
+using System.Data;
+using System.Linq;
+using System.Threading.Tasks;
+using YesSql.Provider.Sqlite;
+using YesSql.Sql;
+using Xunit;
+
+namespace YesSql.Tests
+{
+    public class NamingPolicyTests
+    {
+        [Fact]
+
+        public async Task UseCustomNamingPolicy()
+        {
+            // Arrange
+            var configuration = CreateConfiguration();
+
+            // Act & Assert
+            configuration.NamingPolicy = new SnakeCaseNamingPolicy();
+
+            var store = await StoreFactory.CreateAsync(configuration);
+            using (var connection = store.Configuration.ConnectionFactory.CreateConnection())
+            {
+                connection.Open();
+
+                using (var transaction = connection.BeginTransaction(store.Configuration.IsolationLevel))
+                {
+                    var builder = new SchemaBuilder(store.Configuration, transaction);
+                    builder.CreateTable("UsersInRoles", column => column
+                            .Column<int>(store.Configuration.NamingPolicy.ConvertName("UserId"))
```

Comment:
Need to find a way to apply naming policy nicely with column name(s)

Reply:
User code should not have to deal with it, it should be part of the schema builder logic, the same way it will add the table prefix by itself.